toolchain/check/impl.cpp

```diff
@@ -296,7 +296,16 @@ auto CheckConstraintIsInterface(Context& context, SemIR::InstId impl_decl_id,
     return SemIR::SpecificInterface::None;
   }
 
-  auto identified_id = RequireIdentifiedFacetType(context, *facet_type);
+  auto identified_id = RequireIdentifiedFacetType(context, *facet_type, [&] {
+    CARBON_DIAGNOSTIC(ImplOfUnidentifiedFacetType, Error,
+                      "impl as facet type {0} that cannot be identified",
```

Comment:
I'm unfamiliar with this term, what does it mean to be an "identified" facet type? Is that a different way of saying "resolved"?